src/component/ProjectsPage/AllProjects/AllProjects.jsx

```diff
@@ -1,83 +1,139 @@
-import "./style.css"
-import Project from "./Project/Project"
-import Alia from "../../../assets/images/projectspage/alia.svg";
-import Arctic from "../../../assets/images/projectspage/arctic.svg";
-import Jurni from "../../../assets/images/projectspage/jurni.svg";
-import ShowNxt from "../../../assets/images/projectspage/shownxt.svg";
-import HotDate from "../../../assets/images/projectspage/hotdate.svg";
-import AutoPasser from "../../../assets/images/projectspage/autopasser.svg";
+import './style.css'
```

Comment:
Can you make this page responsive on resize? I think its just on emulator right now

Reply:
hm you mean like vertical scrolling? Im a bit confused. It is responsive